recipe/meta.yaml

```diff
@@ -1,16 +1,37 @@
 {% set name = "usgscsm" %}
-{% set version = "1.7.0" %}
+{% set version = "2.0.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://github.com/USGS-Astrogeology/{{ name }}/archive/{{ version }}.tar.gz
-  sha256: 4a60490399c44de5818e4ae49cd941b03b42cf78464fa84fb0a8ed35f52603c5
+  - folder: .
+    url: https://github.com/DOI-USGS/{{ name }}/archive/{{ version }}.tar.gz
+    sha256: a8ae036caa0b958fa6e28ee8ba7a29c2b6fefeb1413144243d59521f51d1dc12
+  
+  # submodules
+  - folder: ale
+    url: https://github.com/DOI-USGS/ale/archive/a7d8a148f54d5141c63ed45a24a71c890ca11de7.zip
+    sha256: d2cecca296de5c44245f0471cf5a3e835fb4792cc9fba24ab9f29822dc67338f
+  - folder: ale/eigen
+    url: https://gitlab.com/libeigen/eigen/-/archive/3.4.0/eigen-3.4.0.tar.gz
+    sha256: 8586084f71f9bde545ee7fa6d00288b264a2b7ac3607b974e54d13e7162c1c72
+  - folder: ale/json
+    url: https://github.com/nlohmann/json/archive/c6b298799aca23f2b35fc52c1df393afede20c15.zip
+    sha256: 21370dcff5ffb3e58b28f3f2d64a828cbb708cc632231ead9a7184988a0e563c
+  - folder: csm
+    url: https://github.com/USGS-Astrogeology/csm/archive/c78dbba3731432249a75db9fdd7415b292d46715.zip
```

Comment:
This should be DOI-USGS no?

Reply:
We seem to have missed porting csm over to the DOI org. Is it available through code.usgs.gov?

Reply:
I get a 404 when I visit https://github.com/DOI-USGS/csm, perhaps it wasn't moved over?

Reply:
Ohh! It's the CSM, csm: https://github.com/ngageoint/csm is the repository of record.

Reply:
@acpaquette We shouldn't port it, because we are not the maintainers on it. We had a copy because it did not have a home. The link above is the proper home. We just need to be careful to get the correct release.

Reply:
@jlaura I think we made some changes to it that should make there way back into the repo you linked

Reply:
Ya, the forked CSM has:

"This branch is [41 commits ahead of](https://github.com/USGS-Astrogeology/csm/compare/ngageoint%3Acsm%3Amaster...master), [27 commits behind](https://github.com/USGS-Astrogeology/csm/compare/master...ngageoint%3Acsm%3Amaster) ngageoint/csm:master"

Since I am done with the ALE release I can look at this in more detail